### PR TITLE
add nlp score metrics

### PR DESCRIPTION
**Pull request by andrewelizondo**

### CODE REVIEW

File: langkit/nlp_scores.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+lang_config = LangKitConfig()
+_corpus = lang_config.reference_corpus
+_scores = lang_config.nlp_scores
+_rouge_type = "rouge1"
+
+diagnostic_logger = getLogger(__name__)
+
+def register_score_udfs():
+    if _corpus:
+        print(_scores)
```
Comment (FelipeAdachi) [inline langkit/nlp_scores.py:15]: remove print?

**Review by FelipeAdachi — COMMENTED** [review]
I'm not sure I understand how this would work.

Is a single corpus used as reference for every row? Isn't the most usual scenario the case where we have a particular context for each row?

Maybe adding a simple test to show how you envision this being used would help.

**Comment by andrewelizondo** [comment]
@FelipeAdachi the current implementation assumes that you have a static corpus to compare against, ideally in the future we make the corpus a computed column so this can be done dynamically, I added a test in [4974e61](https://github.com/whylabs/langkit/pull/87/commits/4974e61e8756c50be837694ba9100fb998bda117)

**Review by FelipeAdachi — APPROVED** [review]
LGTM!

**Pull request merged**